pkgs/misc/emulators/wine/base.nix

```diff
@@ -115,4 +121,6 @@ stdenv.mkDerivation ((lib.optionalAttrs (! isNull buildScript) {
     description = "An Open Source implementation of the Windows API on top of X, OpenGL, and Unix";
     maintainers = with stdenv.lib.maintainers; [ avnik raskin bendlas ];
   };
-})
+  } // lib.optionalAttrs stdenv.isDarwin {
+    hardeningDisable = [ "fortify" ];
```

Comment:
`hardeningDisable` can go up above with a `lib.optional`. This isn't a mass rebuild, right?